selfdrive/test/test_models.py

```diff
@@ -197,11 +197,12 @@ def test_panda_safety_carstate(self):
 
     checks = defaultdict(lambda: 0)
     for can in self.can_msgs:
-      for msg in can_capnp_to_can_list(can.can, src_filter=range(64)):
+      msgs = can_capnp_to_can_list(can.can, src_filter=range(64))
```

Comment:
Verified through both the CANParser and a full end to end test (sending CAN messages through jungle to panda to boardd to can socket) that the order you send messages in one cycle is the order you get the messages.